src/modules/md/process.cpp

```diff
@@ -308,10 +307,10 @@ bool MDModule::process(Dissolve &dissolve, const ProcessPool &procPool)
         ke *= 0.01;
 
         // Write step summary?
-        if ((step == 1) || (step % outputFrequency_ == 0))
+        if ((step == 1) || (step % outputFrequency_.value_or(0) == 0))
```

Comment:
I have a suspicion that this will write out every step (or maybe even crash!) if `outputFrequency_` is undefined, since the result of `step % 0` is undefined. It's safer to do:
```suggestion
        if (outputFrequency_ && (step == 1 || (step % outputFrequency_.value() == 0)))
```